ls1intum/Artemis
# K6 tests for static code analysis

### Motivation and Context


At the moment K6 tests for programming exercises with SCA are not supported.

### Description

- [x] Add Java SCA test
- [x] Remove XSRF tokens
- [x] Update Java solution template
- [x] Add new stages for Bamboo and Jenkins build plans executing the tests.

### Steps for Testing


1. Code review
2. Check the build plans and run the tests using this branch. Bamboo: https://bamboo.ase.in.tum.de/browse/ARTEMIS-SAT4; Jenkins: https://bamboo.ase.in.tum.de/browse/ARTEMIS-SATJG
